### ISSUE TITLE
improved, common panels

### ISSUE THREAD

**Issue by omerlh** [metalmatze/slo-libsonnet#23]
Following the discussion on kubernetes-monitoring/kubernetes-mixin#326, I think it will be good to create reusable components that can be used on other dashboards - e.g. panels. I was thinking about the following:
 - SLO 
 - Error budget
 - Combination of them both?

It should be probably a single stats, and maybe SLO over time (single stat support showing a little graph under it). The challenge is keeping the alert rules and dashboards the same - so we need to change some alerts to recording rules, and use them both for dashboarding and alerting.

Thoughts?

**Comment by metalmatze** [comment]
The generated error rules and alerts actually have a snippet to also generate Grafana panels: https://github.com/metalmatze/slo-libsonnet/blob/master/slo-libsonnet/errors.libsonnet#L74-L115

We would "just" need to add some that make sense to the other ones like errorburn.
The idea here is, that you just get the individual panels and not a full dashboard, as those panels are most likely integrated into some bigger dashboard in most cases.

**Comment by omerlh** [comment]
Yep, that exactly what I had in mind - and letter on using them as building blocks on the kubernetes mixin project.

**Comment by omerlh** [comment]
Looking back at the code - look like most of the things is there, I was maybe thinking about organizing it so you'll have ready-made SLOs for RED (there is already latency and errors) and USE methods. What do you think? Does this makes sense?
